packages/guideline-blocks-settings/src/components/Attachments/Attachments.spec.ct.tsx

```diff
@@ -103,7 +103,6 @@ describe('Attachments', () => {
         });
 
         if ((await isPre302Stub(appBridge)) && hasOpenAssetChooser(appBridge)) {
-            // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
`pnpm lint:fix` fixed this one

Reply:
Probably not, pretty sure tests will fail with AppBridge 3.0.0

Reply:
yepppp...